src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -74,19 +74,45 @@ def build_string_from_input(prompt, bos_token, image_seq_len, image_token):
     """
     return f"{image_token * image_seq_len}{bos_token}{prompt}\n"
 
+
+class PaliGemmaTextKwargs(TextKwargs):
+    suffix: Optional[Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]]] = None
+
+
+class PaliGemmaImagesKwargs(ImagesKwargs):
+    do_convert_rgb: Optional[bool] = None
+    do_thumbnail: Optional[bool] = None
+    do_align_long_axis: Optional[bool] = None
+    do_rescale: Optional[bool] = None
```

Comment:
`TypedDict` kwarg classes don't need to hold default `None` values or others, they only serve to indicate the accepted args and their type!